libs/components/core/src/lib/modules/content-info-provider/descriptor-type.ts

```diff
@@ -0,0 +1,4 @@
+export enum SkyDescriptorType {
```

Comment:
Did you mean to use this for the descriptor interface's `type` property? If so I recommend using a string union instead of a string enum as those are easier to use in templates.

Reply:
I swear to god I deleted this file before I committed 🤯